src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -5098,6 +5098,106 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     theEmitter->emitIns_R_R_I(INS_sve_uqrshrn, EA_SCALABLE, REG_V15, REG_V12, 1,
                               INS_OPTS_SCALABLE_H); // UQRSHRN <Zd>.H, {<Zn1>.S-<Zn2>.S }, #<const>
 
+    // IF_SVE_DM_2A
+    theEmitter->emitIns_R_R(INS_sve_decp, EA_8BYTE, REG_R0, REG_P0, INS_OPTS_SCALABLE_B); // DECP <Xdn>, <Pm>.<T>
```

Comment:
For consistency with instructions elsewhere, INS_OPTS_SCALABLE_B should be INS_OPTS_SCALABLE_B_WITH_SCALAR. Same for anything else where EA_ is a value instead of  EA_SCALABLE.